docs/DoxygenLayout.xml

```diff
@@ -5,24 +5,22 @@
     <tab type="mainpage" visible="yes" title=""/>
     <tab type="pages" visible="no" title="" intro=""/>
     <tab type="usergroup" visible="yes" title="Tools" intro="Documents" >
-      <tab type="user" url="@ref cyberway-eosio-abidiff" visible="yes" title="CyberWay-Eosio Abidiff" />
-      <tab type="user" url="@ref cyberway-eosio-abigen" visible="yes" title="CyberWay-Eosio Abigen" />
-      <tab type="user" url="@ref cyberway-eosio-cpp" visible="yes" title="CyberWay-Eosio Cpp" />
-      <tab type="user" url="@ref cyberway-eosio-init" visible="yes" title="CyberWay-Eosio Init" />
-      <tab type="user" url="@ref cyberway-eosio-ld" visible="yes" title="CyberWay-Eosio Ld" />
+      <tab type="user" url="@ref eosio-abigen" visible="yes" title="Eosio Abigen" />
+      <tab type="user" url="@ref cyberway-cpp" visible="yes" title="CyberWay Cpp" />
+      <tab type="user" url="@ref eosio-abigen" visible="yes" title="Eosio Abigen" />
```

Comment:
Duplicated.

Reply:
yes, it is my typo. It should be cyberway-abidiff in the 8 line